### PR TITLE
[Enhacement] Allow test.py to save evaluation results

### PR DESCRIPTION
**Pull request by SingleZombie**
## Motivation

The evaluation results of `test.py` are hard to redirect to a file.

## Modification

Use logger to manage all the logs of `test.py`. Now all the outputs of `test.py` will be saved to `--log2file`.

### CODE REVIEW

File: mmdeploy/codebase/mmcls/deploy/classification.py [hunk 1]
```diff
@@ -224,13 +226,19 @@ def evaluate_outputs(model_cfg: mmcv.Config,
                 evaluation. It is useful when you want to format the result
                 to a specific format and submit it to the test server.
                 Default: False.
+            log_file (str | None): The file to write the evaluation results.
+                Defaults to `None` and the results will only print on stdout.
         """
         import warnings
+        from mmcv.utils import get_logger
+        logger = get_logger('test', log_file=log_file, log_level=logging.INFO)
+        print(log_file)
```
Comment (RunningLeon) [inline mmdeploy/codebase/mmcls/deploy/classification.py:235]: Are these two line for debugging?

Comment (SingleZombie) [reply mmdeploy/codebase/mmcls/deploy/classification.py:235]: Line 235 is for debugging! I will delete it.

**Review by RunningLeon — APPROVED** [review]
LGTM

File: tools/test.py [hunk 2]
```diff
@@ -72,7 +71,8 @@ def parse_args():
     parser.add_argument(
         '--log2file',
         type=str,
-        help='log speed in file format, require speed-test first')
+        help='log evaluation results and speed to file',
```
Comment (AllentDan) [inline tools/test.py:74]: may move the arg above --speed-test and update the document accordingly

**Review by AllentDan — APPROVED** [review]

**Pull request merged**